app/src/main/res/values/edit_name_list_strings.xml

```diff
@@ -39,7 +39,11 @@
     <string name="multiple_deletions_title">How many copies of this name do you want to delete?</string>
     <string name="names_deleted">Name(s) deleted.</string>
 
-    <string name="multiple_renames_title">How many copies of this name do you want to change?</string>
+    <string name="multiple_renames_title">
+        How many copies of %s do you want to change?
+         You currently have %d copies of this name.
```

Comment:
nit: Newline in XML will be interpreted as a blank space in rendering IIRC. Removing this should prevent double spacing.
```suggestion
        You currently have %d copies of this name.
```